templates/.circleci/config.yml

```diff
@@ -4,8 +4,12 @@ unit_tests: &unit_tests
     - restore_cache:
         key: dependency-cache-{{ checksum "package-lock.json" }}
     - run:
-        name: NPM Rebuild
-        command: npm install
+        name: Install Latest NPM.
```

Comment:
```
name: Install
```

Can we please do it in one step `npm i -g npm@latest && npm ci` ? I think we don't need the `npm` version check as installing the latest version is best practice anyways

Reply:
@michael-ciniawsky `npm@3` is very buggy and can't be updated to `latest`, we need check.

Reply:
I'm not following why and when would this likely fail ? What is `nvm` doing differently here and do we nee to use it to install `npm@latest` then ?

Reply:
@michael-ciniawsky it is internal bug in `nvm` and `npm`, you can try this, just update CircleCI config in any repo and try to run CI (after update you will get error `Error: Cannot find module 'strip-ansi'`)

Reply:
Ok, no clue :) just leave it for now as is then, not too important